src/app/shared/nav/nav-menu.component.ts

```diff
@@ -1,50 +1,52 @@
-import { Component, OnDestroy } from '@angular/core';
+import { Component, OnDestroy, HostListener } from '@angular/core';
 import { Store, select } from '@ngrx/store';
-import { Observable } from 'rxjs/Observable';
 import { Subscription } from 'rxjs/Subscription';
-import { PodcastModel, EpisodeModel, RouterModel, CHARTTYPE_EPISODES,
-  MetricsType, METRICSTYPE_DOWNLOADS, METRICSTYPE_TRAFFICSOURCES, METRICSTYPE_DEMOGRAPHICS,
-  PodcastPerformanceMetricsModel, EpisodePerformanceMetricsModel } from '../../ngrx/';
-import {  selectSelectedPodcast, selectRecentEpisode, selectRouter,
-  selectSelectedPodcastPerformanceMetrics, selectRecentEpisodePerformanceMetrics } from '../../ngrx/reducers/selectors';
+import { RouterModel, MetricsType, METRICSTYPE_DOWNLOADS, METRICSTYPE_TRAFFICSOURCES, METRICSTYPE_DEMOGRAPHICS } from '../../ngrx/';
+import {  selectRouter } from '../../ngrx/reducers/selectors';
 import * as ACTIONS from '../../ngrx/actions';
 
 @Component({
   selector: 'metrics-nav-menu',
   template: `
-    <metrics-profile
-      [podcast]="selectedPodcast$ | async"
-      [episode]="mostRecentEpisode$ | async"
-      [podcastPerformance]="podcastPerformance$ | async"
-      [episodePerformance]="episodePerformance$ | async"
-      (chartEpisode)="onChartSingleEpisode($event)"></metrics-profile>
-    <nav>
-      <button *ngFor="let t of types"
-          (click)="routeMetricsType(t.type)"
-         [class.active]="routerState.metricsType === t.type">{{t.name}}</button>
-    </nav>
+    <div class="dropdown" [class.open]="open">
+      <div class="overlay" (click)="toggleOpen()"></div>
+      <div class="dropdown-button">
+        <button (click)="toggleOpen()" >{{routerState.metricsType}}<span class="down-arrow"></span></button>
+      </div>
+      <div class="dropdown-content rollout">
+        <nav>
+          <li *ngFor="let t of types">
+            <button
```

Comment:
In mobile view ... when i click/select on these buttons, the dropdown doesn't close.  Have to click outside of the dropdown to make it close.

Reply:
will take a look!